ml-agents-envs/mlagents/envs/sampler_class.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+from functools import *
+from collections import OrderedDict
+from abc import ABC, abstractmethod
+
+from .exception import SamplerException
+
+class Sampler(ABC): 
+
+    @abstractmethod
+    def sample_parameter(self, *args, **kwargs):
```

Comment:
sample_parameter shouldn't take args and kwargs (or should maybe take a context arg if you're designing for the future)

Reply:
type annotation for return type: ` -> float:`

Reply:
And same annotation for all the implementations.